### PR TITLE
setting up index.html file

### PR DESCRIPTION
**Pull request by mateahoward**

### CODE REVIEW

**Comment by mateahoward** [comment]
@Junior-Developer-Group/hacktoberfest-maintainers 

Let's take this as a PR example. 

We have assignees (the one who made PR),
we have requested reviewers (all of you, whoever gets to it first) and 
we have 'hacktoberfest-accepted' label added -- this is added after (or just before) you've merged the PR in but we have to keep that one in mind :) 

I did ask for 'commit messages' to make it easier but let's keep that as somewhat optional :)

**Review by J3ffJessie — APPROVED** [review]
You got the message in the console. Yay!!! Recruiting.

**Pull request merged**
